app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -1,7 +1,13 @@
 class Api::V1::Merchants::RevenueController < ApplicationController
   def show
     merchant = Merchant.find(params[:id])
-    
+
     render json: RevenueSerializer.new(RevenueFacade.merchant_revenue(merchant))
   end
+
+  def index
+    quantity = params[:quantity]
+
+    render json: MerchantSerializer.new(RevenueFacade.merchants_by_revenue(quantity))
```

Comment:
Calls the merchant serializer here due to desired data return, Another skinny controller yay!